modules/apps/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/page_attachment_action.jsp

```diff
@@ -65,18 +74,7 @@ WikiPage wikiPage = WikiPageAttachmentsUtil.getPage(attachmentsFileEntry.getFile
 		</c:when>
 		<c:otherwise>
 			<c:if test="<%= WikiPagePermission.contains(permissionChecker, wikiPage, ActionKeys.DELETE) %>">
-				<portlet:actionURL name="/wiki/edit_page_attachment" var="deleteURL">
-					<portlet:param name="<%= Constants.CMD %>" value="<%= trashHelper.isTrashEnabled(scopeGroupId) ? Constants.MOVE_TO_TRASH : Constants.DELETE %>" />
-					<portlet:param name="redirect" value="<%= currentURL %>" />
-					<portlet:param name="nodeId" value="<%= String.valueOf(wikiPage.getNodeId()) %>" />
-					<portlet:param name="title" value="<%= wikiPage.getTitle() %>" />
-					<portlet:param name="fileName" value="<%= HtmlUtil.unescape(attachmentsFileEntry.getTitle()) %>" />
-				</portlet:actionURL>
-
-				<liferay-ui:icon-delete
-					trash="<%= trashHelper.isTrashEnabled(scopeGroupId) %>"
-					url="<%= deleteURL %>"
-				/>
```

Comment:
Why do we stop using the liferay-ui:icon-delete??

Couldn't we use it like here?: discussion/view_message_thread.jsp:241

I mean. Have a js function that controls how it should behave (calling, for example the logic you introduced in attachmentsList.jsp logic)

Anyway, make this logic consitent with edit_page_attachment.jsp looks good to me